src/test/java/brave/opentracing/OpenTracing0_32_BraveTracerTest.java

```diff
@@ -148,4 +155,19 @@
           .isNull(); // new trace
     }
   }
+
+  @Test public void injectRemoteSpanTraceContext() {
+    BraveSpan openTracingSpan = opentracing.buildSpan("encode")
+        .withTag("lc", "codec")
+        .withTag(Tags.SPAN_KIND, Tags.SPAN_KIND_PRODUCER)
+        .withStartTimestamp(1L).start();
+
+    Map<String, String> map = new LinkedHashMap<>();
+    TextMapAdapter request = new TextMapAdapter(map);
+    opentracing.inject(openTracingSpan.context(), Format.Builtin.HTTP_HEADERS, request);
```

Comment:
this is the major new thing here.. we can do injection formats based on span kind.